queries/encass.gql

```diff
@@ -1,16 +1,13 @@
-query encassByName ($name: String!, $policyName: String!, $includeDirectDependencies: Boolean = false, $includeAllDependencies: Boolean = false) {
+query encassByName ($name: String!, $policyName: String!, $includeAllDependencies: Boolean = false) {
     encassConfigByName (name: $name) {
         {{EncassConfig}}
     }
-    policyFragmentByName(name: $policyName) {
-        {{PolicyFragment:-policy}}
+    policyByName(name: $policyName) {
+        {{L7Policy:-policy,policyRevision,policyRevisions}}
         policy {
             xml
-            directDependencies @include (if: $includeDirectDependencies) {
-                {{PolicyDependency}}
-            }
             allDependencies @include (if: $includeAllDependencies) {
-                {{PolicyDependency}}
+                {{PolicyDependency:-policyFragments,ldaps,fips,fipUsers,fipGroups}}
```

Comment:
why we are excluding ldaps and fips ?

Reply:
as they are deprecated ones, we would like to avoid them wherever it is possible.